newscoop/application/controllers/RegisterController.php

```diff
@@ -137,39 +135,10 @@ public function confirmAction()
         }
 
         $this->view->form = $form;
-        $this->view->img = $this->getUserImageSrc($user);
```

Comment:
it was used only by TW, it was ported from old WOBS, templates have been fixed already to get rid of it and use `$user->image`